index.html

```diff
@@ -17,7 +17,9 @@
         </nav>
     </div>
```

Comment:
- [ ] Kindly make sure your elements are well positioned in the `header`. 
![image](https://user-images.githubusercontent.com/97100091/193916886-cefde102-9be0-4428-845a-541fd58c6cd1.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193917002-50a9c77a-84f3-4e6f-a558-f94791313b0a.png)
- [ ] Kindly you may consider changing the color of your hamburger menu `bars` from black to bleu as provided in the Figma.
![image](https://user-images.githubusercontent.com/97100091/193917322-3d9c0756-61f8-47af-893f-3a509c38fd42.png)  ![image](https://user-images.githubusercontent.com/97100091/193917415-d847e282-864d-4d28-a808-cec5316272c4.png)